### DISCUSSION TITLE
Hints/spoilers

### DISCUSSION THREAD

**Discussion by c0urtastic** [baetheus/bee#59] [Ideas]
Hints before the spoiler showed the full answers - reveal first letter, reveal word length, etc. I read your welcome announcement, but if you have time some day!

**Comment by KittRichards** [comment]
I am a regular hint-writer for The NYT's Spelling 🐝 puzzle.

I would be more than pleased to offer my hint-writing services to the I🐝, if the author would like to d/w me?

**Comment by baetheus** [comment]
Hey @c0urtastic and @KittRichards !

Hints sound great. As you've likely noticed I barely have time to make more puzzles each year. This feature, although I love the idea, likely requires more time than I can put in. But if I do come around to this project in a real way in the future I will definitely reach out. Thanks for the input!
